simtool.h

```diff
@@ -702,6 +705,29 @@ class tool_stop_mover_t : public two_click_tool_t {
 	void read_start_position(player_t *player, const koord3d &pos);
 };
 
+class tool_change_city_of_building_t : public two_click_tool_t {
```

Comment:
Inheriting `two_click_kartenboden_tool_t` allows users to select only the ground tiles.
```suggestion
class tool_change_city_of_building_t : public two_click_kartenboden_tool_t {
```